FluentTerminal.SystemTray/Services/UpdateService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using System;
+using Windows.ApplicationModel;
+
+namespace FluentTerminal.SystemTray.Services
+{
+	public class UpdateService
+	{
+		private const string apiEndpoint = "https://api.github.com";
+
+		public bool GetLatestReleasedVersion()
```

Comment:
From the name, I would expect this method to return some Version information object. Maybe rename to something like 'IsNewerVersionAvailable'

Reply:
Not sure if we should include it in this PR or not, but it would be nice to have the option to see if the newest version is a stable/beta/dev version, and optionally choose to update based on that. That might be a reason to return actual version strings or information.

Reply:
I actually wanted to use this api endpoint: https://developer.github.com/v3/repos/releases/#get-the-latest-release 
But there are only pre-releases currently, as such I don't get anything back from the endpoint.
Switching to this endpoint or including additional information in the message would not be a problem.